cdap-watchdog/src/main/java/co/cask/cdap/metrics/process/MetricsConsumerMetaTable.java

```diff
@@ -47,11 +47,13 @@ public MetricsConsumerMetaTable(MetricsTable metaTable) {
     metaTable.put(updates);
   }
 
-  public synchronized <T extends MetricsMetaKey> void save(T key, byte[] value) throws Exception {
+  public synchronized <T extends MetricsMetaKey> void saveBytes(Map<T, byte[]> offsets) throws Exception {
```

Comment:
Not necessary. Trying to follow the style of other methods. Probably the `save` method above doesn't need to be `synchronized` either?